lib/screens/IngredientScreen.dart

```diff
@@ -618,3 +830,680 @@ class _IngredientPageState extends State<IngredientPage> {
+                    Flexible(
+                      child: Text(
+                        'Search for an ingredient to get started',
+                        style: TextStyle(
+                          fontSize: 32,
+                          color: Colors.black,
+                          fontWeight: FontWeight.w400,
+                        ),
+                        textAlign: TextAlign.center,
+                      ),
+                    ),
+                  ],
+                ),
+              ),
+              Container(
+                width: MediaQuery.of(context).size.width / 1.5,
+                padding: const EdgeInsets.all(5),
+                decoration: const BoxDecoration(
+                  borderRadius: BorderRadius.all(Radius.circular(20)),
+                  color: textFieldBacking,
+                ),
+                child: LayoutBuilder(builder:
+                    (BuildContext context, BoxConstraints constraints) {
+                  return Row(
+                    children: <Widget>[
+                      SizedBox(
+                        width: constraints.maxWidth - topBarIconSize,
+                        child: searchField,
+                      ),
+                      const Icon(
+                        Icons.search,
+                        color: Colors.black,
+                        size: topBarIconSize,
+                      ),
+                    ],
+                  );
+                }),
+              ),
+              Container(
+                width: MediaQuery.of(context).size.width,
+                padding: const EdgeInsets.only(
+                    top: 150, left: 15, right: 15, bottom: 50),
+                child: Row(
+                  children: const <Widget>[
+                    Flexible(
+                      child: Text(
+                        'Or scan a barcode to automatically add it to your inventory',
+                        style: TextStyle(
+                          fontSize: 32,
+                          color: Colors.black,
+                          fontWeight: FontWeight.w400,
+                        ),
+                        textAlign: TextAlign.center,
+                      ),
+                    ),
+                  ],
+                ),
+              ),
+              ElevatedButton(
+                onPressed: () {
+                  //TODO(31): Add ability to scan barcodes
+                },
+                style: ElevatedButton.styleFrom(
+                  shape: RoundedRectangleBorder(
+                    borderRadius: BorderRadius.circular(10),
+                  ),
+                  backgroundColor: mainScheme,
+                  padding:
+                      const EdgeInsets.symmetric(vertical: 15, horizontal: 25),
+                  shadowColor: Colors.black,
+                ),
+                child: const Text(
+                  'Scan!',
+                  style: TextStyle(
+                    fontSize: 50,
```

Comment:
You've already done a good job of creating variables, but I would encourage you to extract all hard coded font/icon sizes into variables for two reasons:
1) easier to change across all your pages
2) encourages you to be intentional about choosing a new size that isn't used elsewhere already

Reply:
I will try to do this in the future, and will change all hardcoded stuff like that over time as I see it.